jetson/face_detector_threaded.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, trained_model, detection_threshold=0.75, cuda=True, set_defau
 
 
         elif ('.pth' in trained_model and 'blazeface' in trained_model):
```

Comment:
what is this checking for specifically and why is `trained_model` not used in the resulting code but the constant `"blazeface.pth"` is?